release-notes/Cube/Cube_Feature_Release_10.5/Cube_10.5.5.md

```diff
@@ -12,49 +12,130 @@ uid: Cube_Feature_Release_10.5.5
 
-## Highlights
+## New features
 
-*No highlights have been selected yet.*
+#### EPM integration now fully available in DataMiner Cube [ID 22125] [ID 24896] [ID 26002] [ID 26232] [ID 29748] [ID 42221]
 
-## New features
+<!-- MR 10.4.0 [CU14] / 10.5.0 [CU2] - FR 10.5.5 -->
 
-#### System Center - SNMP forwarding: New option to prevent an SNMP manager from resending SNMP inform messages [ID 41885]
+All EPM integration functionality that previously required the *CPEIntegration* soft-launch option is now fully available in DataMiner Cube. This allows you to access EPM data directly through the Cube UI, instead of only via an EPM Manager element. The data is available via a dedicated app, but can also be accessed via the Surveyor or the Alarm Console.
 
-<!-- MR 10.4.0 [CU14]/10.5.0 [CU2] - FR 10.5.5 -->
+##### Topology app
 
-Up to now, when you stopped and restarted an SNMP manager, all open alarms would be resent. From now on, when you configure an SNMP manager, you will be able to prevent this by selecting the *Enable tracking to avoid duplicate inform acknowledgments (ACKs)* option. If you select this option, DataMiner will track which inform messages have been sent, and will not resend those that have already been acknowledged.
+The most prominent new EPM integration feature is the Topology app. If your system contains at least one correctly configured EPM Manager element (as detailed below), this app becomes available via a button in the Cube sidebar. Clicking the button will open a pane where you can select the front-end EPM Manager (in case more than one is available), select the topology chain, drill down to any of the topology levels in that chain, and open a card representing the data of the selected item. When you navigate between (docked) EPM cards, the filter selection will be updated to match the displayed card.
 
-> [!NOTE]
-> This new *Enable tracking to avoid duplicate inform acknowledgments (ACKs)* option is not selected by default and is not compatible with the existing *Resend all active alarms every:* option. It is also not compatible with the *Resend...* command, which can be selected after right-clicking an SNMP manager in the *SNMP forwarding* section of System Center.
+![Topology app](~/user-guide/images/EPMIntegration_Topology_app.png)
 
-#### EPM functionality is now fully integrated in DataMiner Cube [ID 42221]
+To make sure the Topology app is available, the following configuration is needed:
 
-<!-- MR 10.4.0 [CU14] / 10.5.0 [CU2] - FR 10.5.5 -->
+1. Make sure the DMS contains at least one front-end EPM Manager element.
+
+   For DataMiner Cube to detect these elements, they should be active, the *type* attribute of the *Display* tag in their connector should be set to "element manager", and they should contain a parameter named *ElementManagerType* set to 1. Also, their name should preferably start with "FE".
```

Comment:
Names starting with "FE" no longer give a technical advantage but can be handy for recognition by the user.